PhotoScraper/RPI_SIS_PhotoScraper.py

```diff
@@ -115,12 +117,21 @@ def saveImagesToFolder(term, course, class_list):
             if k == "img url":
                 img_url = class_list[i].get(k)
         # download and save the image to a specific folder (term/course_section) from the image url
-        img_name = rcs_id+".png"
-        filepath = path / img_name
         if img_url.split("/")[-1].strip() == "web_transparent.gif":
             print("Skipping {} because no photo on SIS".format(rcs_id))
             continue
         r = requests.get(img_url)
+
+        #Open the image use PIL.Image to deduce the extension
+        img_format = imghdr.what(BytesIO(r.content)).lower()
+        img_name = rcs_id
+        if img_format == "jpeg":
+            img_name = img_name + ".jpg"
```

Comment:
Why are we against just naming the file using the valid `.jpeg` ending and not having this if statement at all?

Reply:
I find it annoying to have a mix of images that are .jpg and .jpeg when trying to search, and with the exception of .tiff I'm used to seeing mostly 3-character extensions for images. Given the primary purpose of the scraper is for uploading to Submitty, we can scrap the if statement and use .jpeg if you think that's best.

Reply:
Meh, I don't want to fight about it, I pulled that part out. Any modern system should support four letter extensions and it does make the code cleaner.

Reply:
Yeah, I don't particularly care either, though do favor cleaner code. Having the if statement would also require a comment explaining why it's there or the next person that goes through might remove it (or at least have the same question I did)